src/resources/rmd/execute.R

```diff
@@ -130,13 +134,27 @@ execute <- function(input, format, tempDir, libDir, dependencies, cwd, params, r
     ), name = "tools:quarto")
     source(file.path(resourceDir, "rmd", "ojs_static.R"))
   }
+
```

Comment:
I don't know if we should be injecting an environment right here (as there could be stuff in knitr that breaks when do this). Where do we inject ojs_define and could we do it in the same place? Another approach would be to just add a hidden function to the global env (sort of like `.First` and `.Last`, so it could be like `.QInline` or something like that). I've just seen too many weird issues when you introduce new environment chaining/parenting on top of existing code that was running under a different arrangement.

Reply:
Ok, I agree. `ojs_define` is injected even more crudely, though, directly in the global environment (a few lines above).

Reply:
I made the change on commit `c52c10479`